examples/search/plotting.ini

```diff
@@ -139,14 +139,14 @@ log-dist =
 far-type = exclusive
 
 [plot_foundmissed-sub_mchirp_grad&plot_foundmissed-all_mchirp_grad&plot_foundmissed-summary]
-distance-type = decisive_optimal_snr
+distance-type = max_optimal_snr
```

Comment:
@GarethCabournDavies Can you explain this change?

Reply:
yeah, it is (somewhat) unrelated to this PR, and was missed in #4385, the point is that given singles, the decisive point on whether a signal can be seen is the maximum SNR, not the (second largest) decisive SNR. I probably should do this as a separate PR though